tests/jeos/firstrun.pm

```diff
@@ -119,6 +119,22 @@ sub verify_partition_label {
     script_output('sfdisk -l') =~ m/Disklabel type:\s+$label/ or die "Wrong partion label found, expected '$label'";
 }
 
+sub verify_selinux {
+    if (has_selinux_by_default) {
+        validate_script_output("sestatus", sub { m/SELinux status: .*enabled/ });
+    } else {
+        my $selinux_supported = script_run("grep -qw selinux /sys/kernel/security/lsm") == 0;
+        my $sestatus_available = script_run("which sestatus");
+        if ($selinux_supported && $sestatus_available) {
+            validate_script_output("sestatus", sub { m/SELinux status:.*disabled/ });
+        } elsif (!$selinux_supported) {
+            assert_script_run("! test -d /sys/fs/selinux", fail_message => "SELinux is supported but /sys/fs/selinux is missing");
```

Comment:
```suggestion
            assert_script_run("test -d /sys/fs/selinux", fail_message => "SELinux is supported but /sys/fs/selinux is missing");
```
If you check if the directory is present as the fail message suggests, this test must not be negated

Reply:
Sorry, I mistyped the fail message. :(